db/migrate/20160710021520_create_tasks.rb

```diff
@@ -0,0 +1,15 @@
+class CreateTasks < ActiveRecord::Migration
+  def change
+    create_table :tasks, id: :uuid do |t|
+      t.string :name, null: false
+      t.index :name, unique: true
+      t.text :description
+      t.integer :state, null: false, default: 15
+      t.uuid :project_id, index: true, null: false
```

Comment:
super minor but instead of using the explicit foreign key call below this could be:
`t.references :project, type: :uuid, index: true, foreign_key: true`